app/models/manageiq/providers/openshift/container_manager/refresher.rb

```diff
@@ -3,9 +3,18 @@ module Openshift
     class ContainerManager::Refresher < ManageIQ::Providers::BaseManager::Refresher
       include ::EmsRefresh::Refreshers::EmsRefresherMixin
 
+      KUBERNETES_EMS_TYPE = ManageIQ::Providers::Kubernetes::ContainerManager.ems_type
```

Comment:
@chessbyte it was my suggestion, I thought we could have used it (in some fashion, with some magic) also to reach:

    ManageIQ::Providers::Kubernetes::ContainerManager::Refresher.entities